test/test_irclib.py

```diff
@@ -840,79 +840,85 @@ def setUp(self):
     def tearDown(self):
         ircdb.networks.networks = {}
 
-    def testStsInSecureConnection(self):
+    def _testStsInSecureConnection(self, cap_value):
         self.irc.driver.anyCertValidationEnabled.return_value = True
         self.irc.driver.ssl = True
         self.irc.driver.currentServer = drivers.Server('irc.test', 6697, None, False)
         self.irc.feedMsg(ircmsgs.IrcMsg(command='CAP',
-            args=('*', 'LS', 'sts=duration=42,port=12345')))
+            args=('*', 'LS', 'sts=' + cap_value)))
 
         self.assertEqual(ircdb.networks.getNetwork('test').stsPolicies, {
-            'irc.test': (6697, 'duration=42,port=12345')})
+            'irc.test': (6697, cap_value)})
         self.irc.driver.reconnect.assert_not_called()
 
-    def testStsInSecureConnectionNoPort(self):
+    def testStsInSecureConnectionWithoutPort(self):
```

Comment:
```suggestion
    def testStsInSecureConnectionWithPort(self):
```

Reply:
(The inline diff doesn't show it well but there were two functions named `testStsInSecureConnectionWithoutPort`)